core/src/main/java/com/opensymphony/xwork2/util/AnnotationUtils.java

```diff
@@ -37,13 +40,25 @@
  * @author Rainer Hermanns
  * @author Zsolt Szasz, zsolt at lorecraft dot com
  * @author Dan Oxlade, dan d0t oxlade at gmail d0t c0m
+ * @author Rob Harrop
+ * @author Juergen Hoeller
+ * @author Sam Brannen
+ * @author Mark Fisher
+ * @author Chris Beams
+ * @author Phillip Webb
```

Comment:
I'm not sure if adding those authors here makes any sense, in most cases we (at ASF) trying to remove the `@author` tag also asking those users about this class can confuse them.